src/app/clusters/time-format-localization-server/time-format-localization-server.cpp

```diff
@@ -52,44 +52,41 @@ class TimeFormatLocalizationAttrAccess : public AttributeAccessInterface
     CHIP_ERROR ReadSupportedCalendarTypes(AttributeValueEncoder & aEncoder);
 };
 
+class AutoReleaseIterator
```

Comment:
Followup: this should probably live somewhere central-ish where we define our generic iterator types.

Reply:
I think we would want some generics, did not try for that yet. We do need auto-realeasing really, but should be part of a separate PR.